### DISCUSSION TITLE
How can I use fulltext search in graphql ? (without postresql function)

### DISCUSSION THREAD

**Discussion by kbrkiras** [hasura/graphql-engine#6024] [General]
I want to search name or surname with contains searchText :

$search=> "%Mike Adam%"

example:
_or: [{ name | '  ' | surname : { _ilike: $search } }]

Just I can do this:

_or: [{ name: { _ilike: $search } }, { surname: { _ilike: $search } }]

Can you help me ? is there solution without postresql function?

**Comment by productdevbook** [comment]
+1

**Comment by tirumaraiselvan** [comment]
Maybe this resource helps you? https://hasura.io/blog/full-text-search-with-hasura-graphql-api-postgres/
